spec/mysql2/error_spec.rb

```diff
@@ -34,7 +34,7 @@
       end
     end
 
-    let(:invalid_utf8) { "\xE5\xC6\x7D\x1F" }
+    let(:invalid_utf8) { ["e5c67d1f"].pack('H*').force_encoding(Encoding::UTF_8) }
```

Comment:
I merged this from the other PR, you can rebase to master to include this commit.